docs/source/advanced/children.rst

```diff
@@ -0,0 +1,95 @@
+Children
+========
+
+Dissect also supports the concept of targets within targets, referred to as child targets. For example, when a
+target contains a ``.vmdk`` file within itself, we can tell ``dissect.target`` to load that target from within the
+context of the first target. This can be useful when dealing with hypervisors.
+
+Say, for example, we opened a Hyper-V target locally from ``\\.\PhysicalDrive0``, we can parse the metadata
+in ``ProgramData/Microsoft/Windows/Hyper-V/data.vmcx`` which tells us where all of the virtual machines are stored.
+We can then use these paths and tell ``dissect.target`` to load another target from there. Reading all of these
+files will still happen from ``\\.\PhysicalDrive0``, passing through the various abstraction layers of ``dissect.target``.
+This allows Dissect to read the disks from running virtual machines, regardless of locks the operating has on these files.
+
+Child targets can be anything that Dissect already supports, but we also provide some automatic detection of 
+child targets for certain systems. Automatic child target detection is currently supported for:
+
+* VMware ESXi (also known as vSphere)
+* VMWare Workstation
+* Microsoft Hyper-V 
+* Windows Subsystem for Linux (WSL2)
+* Virtuozzo
+
+Using child targets
+-------------------
+
+To apply ``target-query`` to child targets simply add the ``--children`` flag like this:
+
+.. code-block:: console
+
+    $ target-query /path/to/target -f hostname --children
+
+Replace the example path ``/path/to/target`` with the path to the host image.
+To just query a specific child, simply use the ``--child`` option and provide the path to
+the file within the host target that contains the child target:
+
+.. code-block:: console
+
+    $ target-query /path/to/target -f hostname --child /virtualmachines/host123.vmcx
+
+
+When using ``target-shell`` you can access the child target by using the ``enter`` command:
+
+.. code-block:: console
+
+    $ enter host123.vmcx
+
+
+Child target API
+----------------
+
+
+To obtain a list of nested targets use ``list_children``:
+
+.. code-block:: python
+
+    target = Target.open("host.img")
+    target.list_children()
+
+This will produce a list of records describing the child targets inside:
+
+.. code-block:: console
+
+    <target/child ... type='hyper-v' path='C:\Hyper-V\Virtual Machines\EC04F346-DB96-4700-AF5B-77B3C56C38BD.vmcx'>
+
+You can now open each target by passing the path attribute to the ``target.open_child()`` method:
+
+.. code-block:: python
```

Comment:
Consider expanding upon this example taking account with the code above. To create a more 'code' complete example.

Reply:
At this point the behavior is the same as a regular target so why should I elaborate on that? Any code here would just clutter the general structure of the document.